reactioncommerce/reaction-component-library
# feat: added text overrides on CartSummary

Part of: #409 
Impact: **minor**  
Type: **feature**

## Component
CartItemDetail now has overrides when i18next will be implemented (see #409)

## Breaking changes
No breaking chances, all added fields are optional

## Testing
1. Add a new prop `cartTitleText` to `<CartSummary />` 
2. Add the value `"test"` to `cartTitleText`
3. Label should display "test" when test is given as a value to the prop. When the prop `cartTitleText` is not added, it should default to the values given in default props

## Added props:

```
cartTitleText: PropTypes.string,
freeText: PropTypes.string,
itemLabelText: PropTypes.string,
itemsText: PropTypes.string,
itemsQuantity: PropTypes.number,
orderTotalLabelText: PropTypes.string,
promoCodeText: PropTypes.string,
shippingLabelText: PropTypes.string,
surchargesLabelText: PropTypes.string,
taxLabelText: PropTypes.string,
```
